lambda_backend/route_microservice/src/getRoutesByGym.ts

```diff
@@ -16,6 +16,7 @@ const getRoutesByGym: Handler = async (event: GetRoutesByGymEvent) => {
     TableName: process.env['ROUTE_TABLE_NAME'],
     IndexName: 'gymLocationIndex',
     ConsistentRead: false,
+    ScanIndexForward: false,
```

Comment:
What does this do?

Reply:
Return routes by reversed sorted date (Latest routes first)